ext/standard/tests/general_functions/parse_ini_booleans.data

```diff
@@ -3,8 +3,7 @@ foo = E_ALL E_NOTICE
 error_reporting = E_ALL
 error_reporting1 = E_COMPILE_ERROR|E_RECOVERABLE_ERROR	|E_ERROR|E_CORE_ERROR 	
 error_reporting2 = E_ALL&~E_NOTICE	 	 	
-error_reporting3 =  E_ALL & ~E_NOTICE 	 
-error_reporting4 = E_ALL & ~E_NOTICE | E_STRICT 
```

Comment:
This should be kept with E_STRICT replaced with something else, I think. Possibly just a dummy value (if hex is supported, something like 0x100000), as this is a parsing test.

Reply:
There are a bunch of whitespaces after are those part of the test too?